### ISSUE TITLE
Move code build tool to webpack instead of gulp

### ISSUE THREAD

**Issue by batpad** [IFRCGo/go-frontend#1199]
We use a tool called `gulp` to manage "building" and optimizing the frontend files and code-base. We have used this tool since the beginning of the project without many changes, and it has worked okay. These days, `webpack` has become the more common tool of choice for this task, and now it is often much harder to find plugins and tools that support `gulp`, whereas it is easier to find things compatible with `webpack`.

Speaking with @frozenhelium, it would make their lives much easier if we moved to `webpack` while building the translation framework, since their existing tools and scripts work with `webpack`.

We should evaluate to make sure this does not turn into a rabbit hole, but it seems to me that we don't have anything overly custom in the project, and ideally the transition to `webpack` should be relatively smooth? If yes, and if it saves time on the translations implementation, I would be in favour of this move.

@frozenhelium - let know if this something you'd be able to look into and give a rough estimation of implementation complexity, and any other things we may need to think through / address if we were to make this move.

**Comment by frozenhelium** [comment]
@batpad I've looked briefly into it and tried to use the webpack. It was pretty straightforward and I successfully got the devserver running with webpack. I spent about an hour doing so, and I think 3-4 more hours will enough to transform all the existing features and clean out a bit. [Here's the current progress.](https://github.com/IFRCGo/go-frontend/pull/1214/files) Let's discuss all the features (listed in the PR) that we'd need to implement before proceeding further. So far, it's going really good.
